### ISSUE TITLE
Question re: geo map CSV export

### ISSUE THREAD

**Issue by KevinTCoughlin** [highcharts/export-csv#95]
When exporting a geo map like http://jsfiddle.net/4yq8dpe5/ the CSV contains:

```
"Category","Random data"
0,0
1,1337
2,
3,
4,
5,
6,
7,
8,
9,
10,
11,
12,
13,
14,
15,
16,
17,
18,
19,
20,
21,
22,
23,
24,
25,
26,
27,
28,
```

Where countries without values are included and the category key is equal to their index.
- Is this intentional?
- Is there a way to configure the chart so that the exported CSV format is `{ hc-key: value }`?

Would appreciate any clarification. Thanks!

**Comment by TorsteinHonsi** [comment]
I fixed the missing country names in 3f06ef6d807383. Regarding the missing values, would you expect the null values to be skipped altogether?

**Comment by KevinTCoughlin** [comment]
Awesome, thanks for the fix @TorsteinHonsi. I don't have much of an opinion on handling `null` values as long as its consistent across visualization types. Feel free to close this issue.
